kubectl-plugin/src/main.rs

```diff
@@ -24,6 +29,9 @@ struct CliArgs {
     /// The operation to be performed.
     #[structopt(subcommand)]
     operations: Operations,
+    /// Output Format
+    #[structopt(default_value = "", short = "o", long = "output")]
```

Comment:
Should option be an enum here?
ni: short and long seem to have their default values, you you can just leave them as short, long

Reply:
Done. Thanks